app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
+end
```

Comment:
these helpers are auto-generated and rarely used. if you ever do need a helper method for your views, ApplicationHelper is a decent place for them, but considering the practice of using them is often eclipsed in goodness by other options, a typical app will only have a handful - negating the necessity for individual helpers.